prediction_market_agent/agents/microchain_agent/blockchain/contract_class_converter.py

```diff
@@ -89,13 +90,15 @@ def generate_microchain_class_from_abi_item(
         # If type mapping fails, we exit. Note that structs as input- or output args are not supported.
         for input in abi_item.inputs:
             if not TYPE_MAPPING.get(input.type, None):
-                logger.info(f"Type mapping has failed. Check inputs {abi_item.inputs}")
+                logger.info(
+                    f"Type mapping for {abi_item.name} has failed. Check inputs {abi_item.inputs}"
```

Comment:
Shouldn't it be `error` or at least `warning` if it failed? So that we catch it in logs.

Reply:
sure, switching to warning